astropy/modeling/tests/test_functional_models.py

```diff
@@ -7,6 +7,7 @@
 from numpy.testing import assert_allclose, assert_array_equal
 from .. import models
 from astropy.coordinates import Angle
+from astropy.modeling import fitting
```

Comment:
I realize this had `from astropy.coordinates import Angle` just above, but that's an oversight.  This should use relative imports:
```python
from ...coordinates import Angle
from .. import fitting
```
could you change both of those while you're at it?